src/pocket/PocketSynthetics.ts

```diff
@@ -0,0 +1,70 @@
+import { Resource } from 'cdktf';
+import { Construct } from 'constructs';
+
+import {
+  SyntheticsMonitor,
+  NrqlAlertCondition,
+} from '@cdktf/provider-newrelic';
+
+export interface PocketSyntheticProps {
+  uri: string;
+  verifySsl: boolean;
+  policyId?: number;
+}
+
+const globalCheckLocations = [
+  'AWS_US_WEST_2',
+  'AWS_EU_WEST_2',
+  'AWS_US_EAST_2',
+];
+
+export class PocketSyntheticCheck extends Resource {
+  constructor(
+    scope: Construct,
+    private name: string,
+    private config: PocketSyntheticProps
+  ) {
+    super(scope, name);
+
+    // if policy id not provided use default policy id
+    // policy in another
+    if (this.config.policyId === undefined) {
+      // I wanted to do a terraform lookup but it doesn't run before generating
+      // the cdktf.json file so we have to hardcode the default policy id
+      this.config.policyId = 1707149; // Pocket-Default-Policy
```

Comment:
I think because the JS part runs before the terraform part so I couldn't figure out how to do a terraform data lookup only if the policy id wasn't passed in?